### PR TITLE
Fix EEx support (for elixir/phoenix)

### PR DESCRIPTION
**Pull request by praveenperera**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const HTML_LANGUAGES: string[] = [
   'haml',
   'handlebars',
   'html',
-  'HTML (Eex)',
+  'HTML (EEx)',
```
Comment (bradlc) [inline src/index.ts]: We need `HTML (EEx)` _and_ `HTML (Eex)` to handle both the [`mjmcloug.vscode-elixir`](https://marketplace.visualstudio.com/items?itemName=mjmcloug.vscode-elixir) and [`jakebecker.elixir-ls`](https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls) extensions:

```suggestion
  'HTML (Eex)',
  'HTML (EEx)',
```

Comment (praveenperera) [reply src/index.ts]: Thanks!

**Comment by bradlc** [comment]
Thanks @praveenperera! I will get this released shortly

**Comment by praveenperera** [comment]
@bradlc Does this change also need to be made in the `tailwindcss-language-server` file?

**Comment by dhc02** [comment]
@praveenperera I think this change is all that's needed, although I had to quit and restart VS Code to get it to scan the project for classes and start autocompleting.

**Comment by praveenperera** [comment]
Good to know it’s working for you @dhc02. Can we get this merged in? @bradlc ?

**Comment by frankdugan3** [comment]
@bradlc Any chance we could get a release that includes this merge?

**Comment by joshchernoff** [comment]
Not working in Version: 1.43.0
only have https://marketplace.visualstudio.com/items?itemName=mjmcloug.vscode-elixir enabled which is required for `HTML (Eex)` for adding the HTML_LANGUAGES string

**Pull request merged**